src/app/my-appliances/components/deployments/deployments.component.html

```diff
@@ -38,32 +38,32 @@
                     </div>
                 </td>
                 <td class="text-left">
-                    <div [ngSwitch]="deploy.task_status.status">
+                    <div [ngSwitch]="deploy.latest_task?.status?.status">
```

Comment:
Wasn't a typo but just propagated from earlier code and Celery backend task meta return structure. The whole thing didn't look right to me either so I've updated it now (https://github.com/galaxyproject/cloudlaunch/commit/15aaf7be72c8a45b28cbce4529090e997315a0fc)